phpBB/includes/acp/acp_attachments.php

```diff
@@ -1228,7 +1228,7 @@ function main($id, $mode)
 						'FILETIME'			=> $user->format_date((int) $row['filetime']),
 						'REAL_FILENAME'		=> (!$row['in_message']) ? utf8_basename((string) $row['real_filename']) : '',
 						'PHYSICAL_FILENAME'	=> utf8_basename((string) $row['physical_filename']),
-						'EXT_GROUP_NAME'	=> (!empty($extensions[$row['extension']]['group_name'])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : '',
+						'EXT_GROUP_NAME'   	=> (!empty($user->lang[$extensions[$row['extension']]['group_name']])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : $extensions[$row['extension']]['group_name'],
```

Comment:
@Nicofuma but it seems that the problem comes from the name of the group, I'm not sure for this.

I only reported it, the team member "tower" can not access the tracker, GitHub, phpBB official site, etc., he lives in Cuba and access to Internet is complicated.

@Elsensee , "tower" commented that he look the same problem in 3.1 and 3.2